modelhub/tests_modelhub/functional/modelhub/logistic_regression_test_utils.py

```diff
@@ -30,8 +30,8 @@ def __init__(self, X: bach.DataFrame, y: bach.Series, **kwargs):
         self.X = X
         self.y = y
 
-        self.X_p = self.X.to_pandas()
-        self.y_p = self.y.to_pandas()
+        self.X_p = self.X.to_pandas().sort_index()
+        self.y_p = self.y.to_pandas().sort_index()
```

Comment:
if you agree with suggestion to `fit`. then this could also be adjusted:
```suggestion
        data = X.copy()
        data[y.name] = y
        pdf = data.to_pandas()

        self.X_p = pdf[X.data_columns]
        self.y_p = pdf[y.name]
```